<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hill</class>
 <widget class="QWidget" name="Hill">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>100</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>492</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>492</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Key Size:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="key_size">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_key">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Key</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Crib:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="crib">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="use_key">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>70</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Use Key</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="use_crib">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>70</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Use Crib</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="beginning">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>70</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Beginning</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
